example/ck_tile/17_grouped_gemm/run_grouped_gemm_example.inc

```diff
@@ -53,26 +53,43 @@ int run_grouped_gemm_example_with_layouts(int argc,
         return -1;
     };
 
+    auto valid_params_size = [&](const auto& Ms,
+                                 const auto& Ns,
+                                 const auto& Ks,
+                                 const auto& s_As,
+                                 const auto& s_Bs,
+                                 const auto& s_Ks,
+                                 int group_count) {
+        return !(Ms.empty() || Ns.empty() || Ks.empty() || s_As.empty() || s_Bs.empty() ||
+                 s_Ks.empty()) &&
+               (Ms.size() == Ns.size() == Ks.size() == s_As.empty() == s_Bs.empty() ==
+                s_Ks.empty() == group_count);
+    };
+
     const int group_count = arg_parser.get_int("group_count");
     const int repeat      = arg_parser.get_int("repeat");
     const int warmup      = arg_parser.get_int("warmup");
 
-    std::vector<ck_tile::index_t> Ms;
-    std::vector<ck_tile::index_t> Ns;
-    std::vector<ck_tile::index_t> Ks;
-    std::vector<ck_tile::index_t> stride_As;
-    std::vector<ck_tile::index_t> stride_Bs;
-    std::vector<ck_tile::index_t> stride_Cs;
+    std::vector<ck_tile::index_t> Ms        = arg_parser.get_int_vec("Ms");
+    std::vector<ck_tile::index_t> Ns        = arg_parser.get_int_vec("Ns");
+    std::vector<ck_tile::index_t> Ks        = arg_parser.get_int_vec("Ks");
+    std::vector<ck_tile::index_t> stride_As = arg_parser.get_int_vec("stride_As");
+    std::vector<ck_tile::index_t> stride_Bs = arg_parser.get_int_vec("stride_Bs");
+    std::vector<ck_tile::index_t> stride_Cs = arg_parser.get_int_vec("stride_Cs");
 
-    for(int i = 0; i < group_count; i++)
+    if(!valid_params_size(Ms, Ns, Ks, stride_As, stride_Bs, stride_Cs, group_count))
```

Comment:
Let's use default parameters only when the user really haven't provided any arguments. If the arguments provided by the user are incorrect or incosistent then we shouldn't run with default set but just terminate execution.